extensions/mongodb-client/runtime/src/main/java/io/quarkus/mongodb/impl/ReactiveMongoClientImpl.java

```diff
@@ -0,0 +1,290 @@
+package io.quarkus.mongodb.impl;
```

Comment:
The convention until now was to have the impl classes in the `runtime` package so it should be `io.quarkus.mongodb.runtime`.

Reply:
This module exposes an API, it would be weird to be in a `runtime` quarkus-specific structure. That being said, for implementation, I don't see an issue.

Reply:
Well, this is the convention we decided. I wanted impl/spi packages but got ruled out.

So right now, the conventions we have are:
`io.quarkus.<extension>`: the potential runtime API
`io.quarkus.<extension>.runtime`: everything that is not API (you can have an `impl` subpackage here if you want)
`io.quarkus.<extension>.deployment`: everything from the deployment artifact